### DISCUSSION TITLE
Nitro

### DISCUSSION THREAD

**Discussion by GefJar** [Rapptz/discord.py#6411] [Q&A] [answered]
How can I see if a user have nitro or not?

**Comment by Alawal** [comment, marked as answer]
Hello, 

the **Discord API does not give bots the possibility to use this endpoint**. 
Discord user-accounts have the ability to make this request, so it would be **therotically possible via a selfbot**.

In Discord.py it's defined in the [class Profile](https://discordpy.readthedocs.io/en/rewrite/api.html#profile) > [`discord.Profile`](https://discordpy.readthedocs.io/en/rewrite/api.html#discord.Profile) with e.g. [`premium`](https://discordpy.readthedocs.io/en/rewrite/api.html#discord.Profile.premium). A selfbot could query premium (Discord nitro).
If you try to reach this endpoint with a bot you also get the error that the whole [`Client.fetch_user_profile`](https://discordpy.readthedocs.io/en/latest/api.html?highlight=profile#discord.Client.fetch_user_profile) method is not reachable.

ARealWant

**Comment by 15696** [comment]
Keep in mind you can detect Nitro in some ways like checking if the user's avatar is animated or by their tag. (like 0001)

**Reply by Rapptz** [reply]
Discriminators aren't nitro exclusive.
